### PR TITLE
misc clean-ups

### LINKED ISSUE #1135: Problem with _pages/tutorials/dragonboard410c/README.md
Can we list the content here under the DragonBoard410c documentation section? That way we can remove the /tutorials/ page that currently has no content...

### LINKED ISSUE #1136: Problem with _pages/verticals/README.md
Hi all,

https://www.96boards.org/verticals/

Can we update this section or remove it? We might be able to use some of the content here elsewhere. 

BR,

Kyle

### LINKED ISSUE #1137: Problem with _pages/openhours/impact/README.md
Hi all,

This page should not exist or at least provide some useful information. 

BR,

Kyle

### PR DESCRIPTION
**Pull request by ric96**
- Fixed 3dtof documentation link and added it to front page.
- Openhours link cleaup: removed old bluejeans links
- removed defunct openhours pages. fixes #1137
- removed verticals page. fixes #1136
- removed db410c tutorials page. fixes #1135

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: _pages/openhours_about.md [hunk 1]
```diff
@@ -50,24 +50,19 @@ We look forward to talking with you.
...
-
-Prior to his work with Linaro/96Boards, he worked for the Qualcomm Institute at UCSD creating an extensive online specialization for the DragonBoard 410c. These six courses are hosted by Coursera.org and contain upward of 500 educational videos.
+Sahaj has a background in Computer Applications engineering and have previously worked as an intern at SUSE. Before Linaro, he was a major contributor to the android-rpi, project that aimed to provide AOSP support for the Raspberry Pi, and also was the owner and maintainer of the official forums for the same. In his free time, he likes to Vlog (Video Blog) about his adventures in the world of Open Source and SBCs on his YouTube Channel (Geek Till it Hertz), where he likes to discuss new ARM hardware and unusual operating systems.
 </div>
```
Comment (sgermanserrano) [inline _pages/openhours_about.md]:
```suggestion

```
Duplicated? There is another one towards the bottom of the page rendering [http://staging-96boards-org-1164.websitepreview.linaro.org/openhours/about/](http://staging-96boards-org-1164.websitepreview.linaro.org/openhours/about/)

File: _pages/openhours_about.md [hunk 2]
```diff
@@ -50,24 +50,18 @@ We look forward to talking with you.
...
-
-Prior to his work with Linaro/96Boards, he worked for the Qualcomm Institute at UCSD creating an extensive online specialization for the DragonBoard 410c. These six courses are hosted by Coursera.org and contain upward of 500 educational videos.
-</div>
+Sahaj has a background in Computer Applications engineering and have previously worked as an intern at SUSE. Before Linaro, he was a major contributor to the android-rpi, project that aimed to provide AOSP support for the Raspberry Pi, and also was the owner and maintainer of the official forums for the same. In his free time, he likes to Vlog (Video Blog) about his adventures in the world of Open Source and SBCs on his YouTube Channel (Geek Till it Hertz), where he likes to discuss new ARM hardware and unusual operating systems.
```
Comment (ric96) [inline _pages/openhours_about.md]: Add </div> back
```suggestion
</div>
Sahaj has a background in Computer Applications engineering and have previously worked as an intern at SUSE. Before Linaro, he was a major contributor to the android-rpi, project that aimed to provide AOSP support for the Raspberry Pi, and also was the owner and maintainer of the official forums for the same. In his free time, he likes to Vlog (Video Blog) about his adventures in the world of Open Source and SBCs on his YouTube Channel (Geek Till it Hertz), where he likes to discuss new ARM hardware and unusual operating systems.
```

File: _pages/openhours_about.md [hunk 3]
```diff
@@ -50,24 +50,19 @@ We look forward to talking with you.
 
 # The Team 
 
-<div class="container">
```
Comment (sgermanserrano) [inline _pages/openhours_about.md]: Should be kept

File: _pages/openhours_about.md [hunk 4]
```diff
@@ -50,24 +50,19 @@ We look forward to talking with you.
...
-<div class="row">
```
Comment (sgermanserrano) [inline _pages/openhours_about.md]: should be kept

**Pull request merged**